gapic/generator/generator.py

```diff
@@ -190,36 +196,30 @@ def _generate_samples_and_manifest(
                     sample,
                 )
 
+                snippet_metadata.file = str(pathlib.Path(out_dir) / fpath)
+
+                index.add_snippet(
+                    snippet_index.Snippet(sample, snippet_metadata))
+
         output_files = {
             fname: CodeGeneratorResponse.File(
                 content=formatter.fix_whitespace(sample), name=fname
             )
             for fname, (_, sample) in fpath_to_spec_and_rendered.items()
         }
 
-        # TODO(busunkim): Re-enable manifest generation once metadata
-        # format has been formalized.
-        # https://docs.google.com/document/d/1ghBam8vMj3xdoe4xfXhzVcOAIwrkbTpkMLgKc9RPD9k/edit#heading=h.sakzausv6hue
-        #
-        # if output_files:
-
-        # manifest_fname, manifest_doc = manifest.generate(
-        #     (
-        #         (fname, spec)
-        #         for fname, (spec, _) in fpath_to_spec_and_rendered.items()
-        #     ),
-        #     api_schema,
-        # )
-
-        # manifest_fname = os.path.join(out_dir, manifest_fname)
-        # output_files[manifest_fname] = CodeGeneratorResponse.File(
-        #     content=manifest_doc.render(), name=manifest_fname
-        # )
+        if len(index.metadata_index.snippets) > 0:
+            # NOTE(busunkim): Not all fields are yet populated in the snippet metadata.
+            # Expected filename: snippet_metadata_{metadata_schema_version}_{apishortname}_{apiversion}.json
```

Comment:
Is this comment still relevant, or is it cruft?

Reply:
Yep still relevant, there's a lot of fields that are not yet filled out.